### PR TITLE
Add am62x support

### PR DESCRIPTION
**Pull request by abelino**

### CODE REVIEW

File: nerves_defconfig [hunk 1]
```diff
@@ -27,7 +28,13 @@ BR2_ROOTFS_OVERLAY="${BR2_EXTERNAL_NERVES_PATH}/board/nerves-common/rootfs_overl
 BR2_ROOTFS_POST_BUILD_SCRIPT="${NERVES_DEFCONFIG_DIR}/post-build.sh ${BR2_EXTERNAL_NERVES_PATH}/board/nerves-common/post-build.sh"
 BR2_ROOTFS_POST_IMAGE_SCRIPT="${NERVES_DEFCONFIG_DIR}/post-createfs.sh"
 BR2_LINUX_KERNEL=y
-BR2_LINUX_KERNEL_USE_ARCH_DEFAULT_CONFIG=y
+BR2_LINUX_KERNEL_CUSTOM_GIT=y
+BR2_LINUX_KERNEL_CUSTOM_REPO_URL="https://git.ti.com/git/ti-linux-kernel/ti-linux-kernel.git"
```
Comment (amclain) [inline nerves_defconfig:32]: We can use tarball links to speed up the download and reduce package size.

File: package/ti-k3-boot-firmware/ti-k3-boot-firmware.mk [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+TI_DM_FW_DIR = ti-dm/am62xx
+TI_DM_FW_FILE = ipc_echo_testb_mcu1_0_release_strip.xer5f
+
+UBOOT_MAKE_OPTS += DM=$(BINARIES_DIR)/$(TI_DM_FW_DIR)/$(TI_DM_FW_FILE)
+TI_K3_R5_LOADER_NEXT_MAKE_OPTS += DM=$(BINARIES_DIR)/$(TI_DM_FW_DIR)/$(TI_DM_FW_FILE)
+
+# ${TI_SECURE_DEV_PKG}/scripts/secure-binary-image.sh ${S}/${DM_FW_DIR}/${FW_NAME} ${S}/${DM_FW_DIR}/${FW_NAME}.signed
+
+# Alternative name? am62-main-r5f0_0-fw
```
Comment (amclain) [inline package/ti-k3-boot-firmware/ti-k3-boot-firmware.mk:29]: Is this comment still needed?

File: package/ti-k3-image-gen/ti-k3-image-gen.mk [hunk 3]
```diff
@@ -0,0 +1,53 @@
+################################################################################
+#
+# ti-k3-image-gen
```
Comment (amclain) [inline package/ti-k3-image-gen/ti-k3-image-gen.mk]: This package isn't being used.

**Review by amclain — APPROVED** [review]
Works on my machine! 🎉 

```
Erlang/OTP 26 [erts-14.1.1] [source] [64-bit] [smp:4:4] [ds:4:4:10] [async-threads:1] [jit]

Interactive Elixir (1.15.7) - press Ctrl+C to exit (type h() ENTER for help)
████▄▄    ▐███
█▌  ▀▀██▄▄  ▐█
█▌  ▄▄  ▀▀  ▐█   N  E  R  V  E  S
█▌  ▀▀██▄▄  ▐█
███▌    ▀▀████
am62_app 0.1.0 (0b212286-4f6c-56b5-09f6-8d74a366aa8a) arm am62x
  Serial       : 0000
  Uptime       : 11.430 seconds
  Clock        : 2020-07-25 00:00:00 UTC (unsynchronized)
  Temperature  : 32.8°C

  Firmware     : Valid (A)               Applications : 36 started
  Memory usage : 151 MB (8%)             Part usage   : 459 MB (2%)
  Hostname     : nerves-0000             Load average : 0.54 0.12 0.04


Nerves CLI help: https://hexdocs.pm/nerves/iex-with-nerves.html

Toolshed imported. Run h(Toolshed) for more info.
iex(1)> 
```

File: package/ti-k3-image-gen/ti-k3-image-gen.mk (hunk 3 above)
Comment (abelino) [reply package/ti-k3-image-gen/ti-k3-image-gen.mk]: Removed in a26a807cbd0c749202740293f436960488a02816.

File: nerves_defconfig (hunk 1 above)
Comment (abelino) [reply nerves_defconfig:32]: Will handle this in another PR. There has been more changes in buildroot + ti-kernel that I need to work through.

**Pull request merged**